public/contact.html

```diff
@@ -0,0 +1,175 @@
+            aria-label="Toggle navigation">
+            <span class="navbar-toggler-bar"></span>
+            <span class="navbar-toggler-bar"></span>
+            <span class="navbar-toggler-bar"></span>
+        </button>
+        <div class="collapse navbar-collapse" id="navbarSupportedContent2">
+            <ul class="navbar-nav ml-auto custom-nav">
+                <a href="/governance" class="btn btn-link">Governance</a>
+                <a href="/security" class="btn btn-link ">IT & Security</a>
+                <a href="/customers" class="btn btn-link ">Customers</a>
+                <a href="/faq" class="btn btn-link ">FAQ</a>
+                <a href="/about-us" class="btn btn-link ">About Us</a>
+                <a href="/contact" class="btn btn-link ">Contact</a>
+            </ul>
+        </div>
+    </div>
+</nav>
+
+<div class="feature-1 section cd-section" id="contact">
+    <div class="container">
+        <div class="row pt-5">
+            <div class="col-md-12 ml-auto mr-auto">
+                <div class="no-transition align-self-end">
+                    <h2 class="title text-center text-uppercase">Contact Us</h2>
+                    <div class="row">
+                        <div class="col-md-4 ml-auto">
+                            <div class="card-body">
+                                <div class="info info-horizontal">
+                                    <div class="icon icon-info">
+                                        <i class="nc-icon nc-chat-33" aria-hidden="true"></i>
+                                    </div>
+                                    <div class="description">
+                                        <h4 class="info-title" style="color: #000000 !important;">Find us at
+                                            the office</h4>
+                                        <p style="color: #000000 !important;"> Dr Tanslaan 12,
+                                            <br> 6229 ET Maastricht,
+                                            <br> The Netherlands
+                                        </p>
+                                    </div>
+                                </div>
+                                <div class="info info-horizontal">
+                                    <div class="icon icon-info">
+                                        <i class="nc-icon nc-badge" aria-hidden="true"></i>
+                                    </div>
+                                    <div class="description">
+                                        <h4 class="info-title" style="color: #000000 !important;">Give us a
+                                            call</h4>
+                                        <p style="color: #000000 !important;">
+                                            Mon - Fri, 9:00-17:00 CET<br/><br/>
+                                            NL: 06 149 27 142<br/>
```

Comment:
@tim-hendriks That's my personal number. Should we remove it? I've only put it in (temporarily) because one organization mandated they could validate our credentials with a phone number on our website.

Reply:
I'll remove that section then, assuming we don't have a business phone number for MDW?